core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -73,6 +71,14 @@ public void addAdapterConfiguration(final Map<String, String> adapterConfigurati
         adapterConfigurations.add(adapterConfiguration);
     }
 
+    /**
+     * Creates a {@link TenantObject} for a tenantId and the enabled property.
```

Comment:
There really is no need to link to the return type. The JavaDoc tool will add links to the return types of all methods by default.